client/my-sites/domains/domain-management/edit/transfer.jsx

```diff
@@ -71,13 +71,20 @@ class Transfer extends React.PureComponent {
 				<Card compact={ false } highlight={ 'warning' }>
 					<div>
 						<h2 className="edit__transfer-text-fail">
-							{ translate( 'Important: Start Your Domain Transfer' ) }
+							{ translate(
```

Comment:
Unrelated, mind adding a `margin-bottom` to this element. It looks weird:
![transfer weird start](https://user-images.githubusercontent.com/1103398/36047357-27a386d0-0ddc-11e8-8aab-e327126e180d.png)

Reply:
1em margin added.
<img width="755" alt="domain_settings_ _site_title_ _wordpress_com" src="https://user-images.githubusercontent.com/1379730/36049058-66465700-0daf-11e8-9ab0-c722fd50f8b2.png">